Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -7445,6 +7447,11 @@ void WebPageProxy::processIsNoLongerAssociatedWithPage(WebProcessProxy& process)
     protectedNavigationState()->clearNavigationsFromProcess(process.coreProcessIdentifier());
 }
 
+void WebPageProxy::isNoLongerAssociatedWithRemotePage(RemotePageProxy&)
+{
+    internals().updatePlayingMediaDidChangeTimer.startOneShot(0_s);
```

Comment:
I added this timer to coalesce calls to `updatePlayingMediaDidChange` if a bunch of remote pages are destroyed during the same event loop turn.

Possibly an unnecessary optimization.